_docs/how-to/staking.md

```diff
@@ -0,0 +1,98 @@
+---
+title: FIO Staking Developer Guide
+description: FIO Staking Developer Guide
+redirect_from:
+    - /docs/integration-guide/staking
+---
+
+# FIO Staking Developer Guide
+## Overview
+[FIP-21](https://github.com/fioprotocol/fips/blob/master/fip-0021.md) is introducing **FIO Staking**, an on-chain program which rewards users for participating in blockchain governance.
+
+This guide is intended to help integrators with FIO Staking integration into their products.
+
+## How FIO Staking works
+### High level
+* User has to vote, proxy, or [auto-proxy](https://developers.fioprotocol.io/docs/how-to/governance) their FIO Tokens.
```

Comment:
For internal devhub references is it is better to use a relative link:

[auto-proxy]({{site.baseurl}}/docs/how-to/governance)

That way the site is not tied to a specific host/domain.